stacks-signer/src/runloop.rs

```diff
@@ -306,8 +313,11 @@ impl RunLoop {
             self.state = State::Uninitialized;
             return Err(ClientError::NotRegistered);
         }
+        if self.state != State::Initialized {
+            // only info log if the state wasn't already initialized before.
+            info!("Runloop successfully initialized!");
+        }
```

Comment:
Same here -- this is a very chatty log line otherwise, and it seems like this isn't really very informative unless the state was actually uninitialized before this call.